### PR TITLE
feature/template-maintenance

### PR DESCRIPTION
**Pull request by cutoffthetop**
# Changes
- clarify release step names
- make python bound order consistent with deps
- add PNGs to ignore files (for playwright testing)
- update versions

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,7 @@ pip-log.txt
 pip-delete-this-directory.txt
 
 # Testing / coverage reports
+tests/
```
Comment (rababerladuseladim) [inline .gitignore:41]: doesn't this ignore all changes to tests?

File: mex-{{ cookiecutter.project_name }}/.gitignore [hunk 2]
```diff
@@ -38,6 +38,7 @@ pip-log.txt
 pip-delete-this-directory.txt
 
 # Testing / coverage reports
+tests/
```
Comment (rababerladuseladim) [inline mex-{{ cookiecutter.project_name }}/.gitignore:41]: same as above: doesn't this ignore all changes to tests?

**Review by rababerladuseladim — CHANGES_REQUESTED** [review]
Only one question.

File: .gitignore (hunk 1 above)
Comment (cutoffthetop) [reply .gitignore:41]: sorry, yeah. that was a copy&paste mistake from the dockerignore

File: mex-{{ cookiecutter.project_name }}/.gitignore (hunk 2 above)
Comment (cutoffthetop) [reply mex-{{ cookiecutter.project_name }}/.gitignore:41]: fixed

**Review by rababerladuseladim — APPROVED** [review]

**Pull request merged**